tests/unit/lms/services/youtube_test.py

```diff
@@ -2,25 +2,87 @@
 
 import pytest
 
-from lms.services.youtube import YoutubeService, factory
+from lms.services.youtube import VideoNotFound, YoutubeService, factory
+from tests import factories
 
 
 class TestYoutubeService:
-    @pytest.mark.parametrize("enabled", (True, False, None))
-    def test_enabled(self, enabled):
-        svc = YoutubeService(enabled=enabled)
+    @pytest.mark.parametrize(
+        "enabled,api_key,expected_enabled",
+        (
+            (True, "api_key", True),
+            (True, "", False),
+            (False, "api_key", False),
+            (False, "", False),
+            (None, "api_key", False),
+            (None, "", False),
+        ),
+    )
+    @pytest.mark.usefixtures("http_service")
```

Comment:
When defined as a parameter pytest does some magic and "uses" the fixture.

`pytest.mark.usefixtures` is only needed if you want to use it (for its side effects for example) but it's not defined as a parameter.

You could remove pytest.mark.usefixtures

Reply:
Oh, I thought it had to be explicitly added for the parameter to end up "being" the fixture. Thanks for the tip.